stand-alone/loaderHelper.ts

```diff
@@ -184,3 +184,14 @@ export const canPlayerRender = (context: RedVenturePlayerContextProps): boolean
 
 	return true;
 };
+
+const newsAndRatingsNamesWithDefaultPlayer = [ 'gamespot', 'gamefaqs', 'comicvine', 'tvguide', 'metacritic' ];
+export const assurePlayerUrl = (context: RedVenturePlayerContextProps): RedVenturePlayerContextProps => {
+	const newContext = { ...context };
```

Comment:
Makes sense to me. We could do the copy in the if statement, and return the new object with the newly set playerUrl only if we get to the `if` statement. Otherwise, we can just return the old untouched context object, in cases of 'giantbomb' for example.

I'm good with the approach you have as well 👍

Reply:
He he, that was my original code and I decided to change it since IMHO it was less readable 😊